packages/components/src/components/as-histogram-widget/as-histogram-widget.e2e.ts

```diff
@@ -91,6 +91,30 @@ describe('as-histogram-widget', () => {
 
       expect(actual.innerText).toEqual('Radio Gaga');
     });
+
+    it('should format X axis ignoring decimals and SI prefixes when range is higher than 1', async () => {
+      const element: E2EElement = await page.find('as-histogram-widget');
+
+      element.setProperty('data', histogramDataFloatingToDecimal);
+      await page.waitForChanges();
+
+      const ticks = await page.findAll('.x-axis text');
+      const ticksValues = ticks.map((tick) => tick.innerText);
+
+      expect(ticksValues).toEqual(['-0.12', '0.88', '1.9', '2.9', '3.9']);
+    });
+
+    it('should format X axis using SI prefixes when range is lower than 1', async () => {
+      const element: E2EElement = await page.find('as-histogram-widget');
+
+      element.setProperty('data', histogramDataFloatingToSIPrefix);
+      await page.waitForChanges();
+
+      const ticks = await page.findAll('.x-axis text');
+      const ticksValues = ticks.map((tick) => tick.innerText);
+
+      expect(ticksValues).toEqual(['120m', '320m', '520m', '710m', '910m']);
```

Comment:
I guess it is, but has this formatting logic been proposed / validated somehow on 'the-product-side'?

Reply:
👌 I'm ok with it. I know formatting has been a source of many questions and requests in the past, so I just wanted to be sure that we were aware of it and agreed on it from the client-side perspective.